modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseActionsInfo.js

```diff
@@ -37,18 +37,22 @@ const BaseActionsInfo = ({
 	setScript,
 	updateActionInfo,
 }) => {
+	if (
```

Comment:
And I also didn't understand why the onAssignment insertion does not occur in the rendering of the page when it is inside useEffect, we would have to fill in the previous fields in order to visualize the onAssignment.